source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
+- **Decompose:**
+Chop up any stories that are too big to handle in a sprint into smaller bits.
+- **Define 'Done':**
+Ensure each story has a clear _Definition of Done_ (DoD) to know exactly when it is crossed the finish line.
+
+**Output:**
+
+- **Refined backlog:**
+Stories that are fleshed out, estimated, and ready for the PM to prioritize
+- **Scored stories:**
+Each story gets a score so we can try to estimate how many stories we should take on based on our prior rate of work.
+- **Approachable tickets:**
+All tickets that go through refinement should have all the info required for any member of the team to pick up.
+
+#### Audience
+
+Data Catalogue Team
+
+### Peer Review
+
+- In the middle of each sprint we have a thirty-minute peer review session
+that provides an opportunity to receive feedback on the current state
+and future plans for the service.
+- [TODO: peer review content]
+
+#### Audience
+
+All of Data Platform and wider audience
+
+### Sprint Retrospective
+
+At the end of the sprint, we:
+
+- Reflect on the sprint’s process, celebrating successes and identifying opportunities for improvement
+- Review the process, not the stories
+- Define how we could improve
+- Share kudos and feedback
+
+#### Audience
+
+Data Catalogue Team
+
+### Sprint Planning
+
+**Purpose:**
+
+Sprint Planning is where we figure out what work we want to commit to for the next sprint. It's where we:
+
+- Discuss and align on the goals for the upcoming sprint
+- Select stories from the refined backlog, ensuring a shared understanding of scope and objectives
+- Define the sprint goal
+- Define the sprint backlog
+
+**Process:**
+
+- **Discussion:** Discuss the stories in the refined backlog, ensuring a shared understanding of scope and objectives, including any work carried from last sprint
+- **Highlighting:** Highlight stories and spikes that are significant to delivery of the sprint goal
+- **Capacity Estimation:** Estimate the capacity of the team for the sprint, and assess whether the tickets brought in match that
+- **Sprint Backlog:** Select the stories that will be brought into the sprint, and assign them to team members where appropriate
+
+**Output:**
+
+- Ensure everyone has stories that they feel they can work on
+- Ensure everyone has a shared understanding of the tasks to deliver this sprint
+
+#### Audience
+
+Data Catalogue Team
+
+## GitHub Practices
+
+- We use GitHub for managing code, documentation, and support.
+- We use GitHub Projects for tracking work and GitHub pull requests for reviewing and merging code.
+- When working in our repos, please adhere to GitHub housekeeping guidelines when creating pull requests,
+ensuring clear titles and descriptions, linking to issues when applicable,
+and requesting reviews upon readiness
+
+### GitHub Housekeeping
```

Comment:
Would this fit better to a 'contributing.md' doc?

Reply:
Page created and content moved